samples/package.json

```diff
@@ -14,7 +14,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@googleapis/iam": "^0.2.0",
-    "google-auth-library": "^7.3.0",
+    "google-auth-library": "^7.1.1",
```

Comment:
this should remain at `7.3.0`, if you need to test samples you can run `npm link ../` within the samples directory to link the current version of the library.